.github/dependabot.yml

```
@@ -48,4 +48,37 @@ updates:
+          - "vite-tsconfig-paths"
+          - "@vitejs/plugin-react"
+      React:
+        patterns:
+          - "react"
+          - "react-dom"
+          - "@types/react"
+          - "@types/react-dom"
+      Typescript:
+        patterns:
+          - "typescript"
+ 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)